server/src/test/java/se/fortnox/reactivewizard/server/RwServerTest.java

```diff
@@ -204,6 +205,21 @@ void shouldAwaitShutdownDependency() {
         ));
     }
 
+    @Test
+    void shouldAwaitShutdownLogging() {
```

Comment:
No test for the timeout logic?

Reply:
Will do 👍

Reply:
The timeout was removed. Nothing to test.